students/10th/hogeunoh/westargram/user/models.py

```diff
@@ -0,0 +1,9 @@
+from django.db import models
+
+
+class Users(models.Model):
+	name		= models.CharField(max_length =50)
+	email		= models.CharField(max_length =50)
+	pw			= models.CharField(max_length =100)
```

Comment:
= 기준으로 줄 맞춰 주세요
변수명도 password로 명확하게 써주세요.
```suggestion
	pw         = models.CharField(max_length =100)
```

Reply:
변경했습니다